spec/rspec/matchers/built_in/yield_spec.rb

```diff
@@ -67,6 +67,23 @@ def each_arg(*args, &block)
       expect(val).to be_nil
     end
 
+    it 'raises an error if given an invalid count argument' do
+      expect { yield_control.exactly('2') }.to raise_error(ArgumentError)
+      expect { yield_control.at_least(:trice_with_typo) }.to raise_error(ArgumentError)
+      expect { yield_control.at_most(nil) }.to raise_error(ArgumentError)
+    end
+
+    it 'is yet to support multiple calls to compatible count constraints' do
+      pending
+      expect { |b| 1.upto(4, &b) }.to yield_control.at_least(3).at_most(4).times
+      expect { |b| 1.upto(2, &b) }.not_to yield_control.at_least(3).at_most(4).times
+    end
+
+    # TODO: This needs implementing.
```

Comment:
Leftover TODO. Don't think `.once.twice` is possible to implement in a reasonable way :D

Reply:
Oops, fixed.